providers/irdma/umain.h

```diff
@@ -0,0 +1,204 @@
+/* SPDX-License-Identifier: GPL-2.0 or Linux-OpenIB */
+/* Copyright (C) 2019 - 2020 Intel Corporation */
+#ifndef IRDMA_UMAIN_H
+#define IRDMA_UMAIN_H
+
+#include <inttypes.h>
+#include <stddef.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <infiniband/driver.h>
+
+#include "osdep.h"
+#include "irdma.h"
+#include "defs.h"
+#include "i40iw_hw.h"
+#include "status.h"
+#include "user.h"
+
+#define PFX	"libirdma-"
```

Comment:
You are not using it

Reply:
Hi Leon,

What do you prefer? I could send a new PR with changes addressing your comments because Shiraz is on vacation or I could send a follow up patch after this PR?

Thank you,
Tatyana

Reply:
Please push new PR with updated kernel-headers, because for this PR I see "This branch has conflicts that must be resolved", and I'll merge it 

Thanks

Reply:
I created a new PR. Sorry for the delay, the 4th of July week was short for us.

Thank you, Tatyana